backend/mocked-api/models/Problem.js

```diff
@@ -0,0 +1,30 @@
+const { Schema, model } = require('mongoose');
+
+const ProblemSchema = new Schema({
+
+    title: {
+        type: "String",
+        required: true
+    },
+    level: {
+        type: Number, min: 1, max: 10,
+        required: true
+    },
+    description: {
+        type: String,
+        required: true
+    },
+    author: {
+        nick: {type: String},
+        email: {type: String}
+    },
+    photo: {type: String},
+    samples: {type: Array, required: true}
```

Comment:
```suggestion
    samples: {type: Array, required: true}
```
É interessante especificar o que uma sample deve ter e.g. entrada, saída, etc.

Reply:
ok